admin-site/src/Occasion.svelte

```diff
@@ -32,11 +32,33 @@
   let formattedStartTime;
   let formattedEndTime;
 
-  //double check that a delete Occasion has happened
-  let deleteOccasionHasHappened = false;
-
   let cueState = 0;
 
+  let mediaDomainEnum = {
```

Comment:
Declaring this with `const` rather than `let` would be a clearer signal of intent.

Reply:
Implemented